cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -3086,8 +3287,10 @@ bool CacheAllocator<CacheTrait>::moveForSlabRelease(
        ++itemMovingAttempts) {
     stats_.numMoveAttempts.inc();
 
-    // Nothing to move and the key is likely also bogus for chained items.
+    // Nothing to move - in the case that tryMoving failed
```

Comment:
Would it be possible to add support for SlabRelease in a separate PR? I think if we leave the current logic (just return false for moveForSlabRelease for chained item, everything should work fine (chained items will always be evicted). 

It would make reviewing (and testing) simpler.